### PR TITLE
[student-libraries] Removed/disabled unimplemented components.

### PR DESCRIPTION
**Pull request by jmkulwik**
# Description
Disable the following: “See Examples” “See More Details” “:arrows_counterclockwise:” button

These were causing a lot of confusion since they aren't yet implemented. They need to be removed until they are officially enabled for student libraries.

image

image

## Links

- [spec](https://docs.google.com/document/d/1VxMmdHdIeBWCXKofkjEy9MuWcrzeEW3Zt-Ir8pINYdA/edit)
- [jira](https://codedotorg.atlassian.net/browse/STAR-710)
- [all relevant PRs](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Ajmkulwik+archived%3Afalse+student+libraries)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/blockTooltips/DropletFunctionTooltip.js [hunk 1]
```diff
@@ -65,7 +65,9 @@ var DropletFunctionTooltip = function(appMsg, definition) {
 
   var localizedDescription = this.getLocalization(this.descriptionKey());
   if (definition.comment) {
+    // debugger;
```
Comment (ajpal) [inline apps/src/blockTooltips/DropletFunctionTooltip.js]: nit: delete

File: apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx [hunk 2]
```diff
@@ -116,11 +120,15 @@ export default class LibraryManagerDialog extends React.Component {
       );
     }
     return libraries.map(library => {
+      let onRefresh;
+      if (this.shouldRefresh(library)) {
```
Comment (ajpal) [inline apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx]: Why this always-false conditional instead of just removing the prop for now?

Comment (jmkulwik) [reply apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx]: Good point. Fixed.

File: apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx [hunk 3]
```diff
@@ -120,7 +116,7 @@ export default class LibraryManagerDialog extends React.Component {
         <LibraryListItem
           key={library.name}
           library={library}
-          onRefresh={this.refreshLibrary}
+          onRefresh={undefined}
```
Comment (ajpal) [inline apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx:131]: nit: cleaner to just remove this line?

Comment (jmkulwik) [reply apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx:131]: I'd prefer to leave it here - helps me remember where it needs to be called for when I implement onRefresh.

**Review by ajpal — APPROVED** [review]

**Pull request merged**